### PR TITLE
Fix missed entities from rector

### PR DESCRIPTION
**Pull request by mabumusa1**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:

This PR continues Rector work, in some parts rector was not able to detect bundles and covert them to classes, so I made this manually. this PR changes the string names of bundles to classes

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. No tests, just code review

### CODE REVIEW

File: app/bundles/ChannelBundle/Event/ChannelEvent.php [hunk 1]
```diff
@@ -71,8 +71,10 @@ public function getRepositoryName($channel)
 
         // if not defined, try the classic naming convention
         $channel = ucfirst($channel);
+        $class   = "\Mautic\\{$channel}Bundle\Entity\\{$channel}";
```
Comment (mabumusa1) [inline app/bundles/ChannelBundle/Event/ChannelEvent.php:74]: @escopecz is this a good approach, or do you have another one?

**Review by daniellord32 — DISMISSED** [review]
Looks good to me

**Review by volha-pivavarchyk — DISMISSED** [review]
Looks good. 

@mabumusa1  Just a question. On the one hand the fully qualified class name, including the namespace, makes the code more readable. But there are a lot of cases when the namespaces are already imported? Can the full calss name be replaced by the short class name, e.g. `\Mautic\LeadBundle\Entity\Lead::class` -> `Lead::class`. WDYT?

File: app/bundles/ChannelBundle/Event/ChannelEvent.php (hunk 1 above)
Comment (escopecz) [reply app/bundles/ChannelBundle/Event/ChannelEvent.php:74]: Yep, let's go with this for now.

File: app/bundles/ChannelBundle/Event/ChannelEvent.php [hunk 2]
```diff
@@ -71,8 +71,10 @@ public function getRepositoryName($channel)
...
+        $repo    = new $class();
```
Comment (escopecz) [inline app/bundles/ChannelBundle/Event/ChannelEvent.php]: But why do we instantiate the class and not passing the FQCN directly?

Comment (mabumusa1) [reply app/bundles/ChannelBundle/Event/ChannelEvent.php]: The functions that calls this function expects the result to be class name not a class instance

**Comment by mabumusa1** [comment]
> Looks good. 
> 
> @mabumusa1  Just a question. On the one hand the fully qualified class name, including the namespace, makes the code more readable. But there are a lot of cases when the namespaces are already imported? Can the full calss name be replaced by the short class name, e.g. `\Mautic\LeadBundle\Entity\Lead::class` -> `Lead::class`. WDYT?

@volha-pivavarchyk thanks for the review, I agree with you on this we will run a rector rule soon to do this

File: app/bundles/ChannelBundle/Event/ChannelEvent.php (hunk 2 above)
Comment (escopecz) [reply app/bundles/ChannelBundle/Event/ChannelEvent.php]: I meant doing directly this:
```php
return "\Mautic\\{$channel}Bundle\Entity\\{$channel}";
```

Comment (mabumusa1) [reply app/bundles/ChannelBundle/Event/ChannelEvent.php]: I tried this before, it should return `class-literal` like this 

```
return "\Mautic\\{$channel}Bundle\Entity\\{$channel}::class";
```

Put this syntax does not work

Comment (escopecz) [reply app/bundles/ChannelBundle/Event/ChannelEvent.php]: Is it PHPSTAN that is causing the trouble?
How about
```php
$class = "\Mautic\\{$channel}Bundle\Entity\\{$channel}";
\assert(class_exists($class));

return $class;
```

Comment (mabumusa1) [reply app/bundles/ChannelBundle/Event/ChannelEvent.php]: Brilliant, that worked check commit b18d35038222be9d3ef0cedfeb5fa9223d4fc696

**Review by escopecz — APPROVED** [review]
Thanks! 👍

**Pull request merged**